clang/lib/Driver/Driver.cpp

```diff
@@ -86,8 +86,10 @@
 #include "llvm/Support/ErrorHandling.h"
 #include "llvm/Support/ExitCodes.h"
 #include "llvm/Support/FileSystem.h"
+#include "llvm/Support/FileUtilities.h" // INTEL
```

Comment:
Do you need the customization comment here since you're adding this to open-source intel/llvm?

Reply:
Doh!  I guess I forgot where I was making changes :)